### ISSUE TITLE
Issue Description: Need to update SPCopy Destination for SharePoint library Items.

### ISSUE THREAD

**Issue by anmolsinghchauhan** [SharePoint/sp-dev-docs#4879]
In My Organization in SharePoint 2010 Send to Copy functionality is used which creates link for manage copies. Now i want to know if we can copy these into links(Send to Copy) into SharePoint Online or create these links into SharePoint online using powershell

**Comment by andrewconnell** [comment]
Sorry... unclear as to what you're asking. Can you rephrase it with a bit more context & detail? Thanks!
